# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libdomainkeys/Makefile.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig       2006-01-18 02:28:58.000000000 +0200
+++ ./Makefile    2009-10-15 17:15:00.000000000 +0300
@@ -3,7 +3,7 @@
 #CFLAGS += -DDK_DEBUG -DDK_HASH_BUFF -Wall
 #CFLAGS += -DUNIXWARE
 INCS=-I.
-LIBS=-L. -ldomainkeys -lcrypto
+LIBS=-L. -ldomainkeys -lcrypto -lresolv
 MAKE=make

